File: ProjectEuler7/ProjectEuler7.java
public class ProjectEuler7{
	public boolean isPrime(long number){
		int root = (int)Math.pow(number, 0.5) + 1;
		for (int i = 2; i < root; i++){
			if (number % i == 0){
				return false;
			}
		}
		return true;
	}

	public static void main(String[] args){
		int count = 0;
		long firstNumber = 1;
		ProjectEuler7 call = new ProjectEuler7();
		while (count < 10002){
			if (call.isPrime(firstNumber)){
				count += 1;
			}
			firstNumber += 1;
		}
		firstNumber -= 1;							//since first number value is still incremented by one at the end so remove that extra one
		System.out.println(firstNumber);
	}
}

File: ProjectEuler7/ProjectEuler7.py
def isPrime(number):
	for i in range(2, (int(number ** 0.5) + 1)):
		if number % i == 0:
			return False
	return True

def main():
	count = 0
	firstNumber = 1
	while count < 10002:
		if isPrime(firstNumber):
			count += 1
		firstNumber += 1
	firstNumber -= 1
	return firstNumber

print(main())